cmd/nerdctl/builder/builder_build_test.go

```diff
@@ -89,17 +89,17 @@ CMD ["echo", "nerdctl-build-test-string"]`, testutil.CommonImage)
 				Expected: test.Expects(0, nil, test.Equals("nerdctl-build-test-string\n")),
 			},
 			{
-				Description: "Successfully build with output docker, name cannot be used",
```

Comment:
@apostasie I've removed this test for now. I may be missing something. What is this testing?